product_template_tags/models/product_template_tag.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProductTemplateTag(models.Model):
+
+    _name = 'product.template.tag'
+    _description = 'Product Tag'
+
+    name = fields.Char(string="Name", required=True, translate=True)
+    color = fields.Integer(string="Color Index")
```

Comment:
Where this field is used ? (I mean, how can we change the value ?)

Reply:
When you select a tag on a product, if you click on the selected tag, you will be able to change the color.

Reply:
Ow. Thanks ! I did'nt know.